applications/nrf_desktop/src/modules/qos.c

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright (c) 2018 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#include <zephyr.h>
+#include <zephyr/types.h>
+
+#include <bluetooth/bluetooth.h>
+#include <bluetooth/gatt.h>
+
+#include "chmap_filter.h"
+
+#include "ble_event.h"
+
+#define MODULE qos
+#include "module_state_event.h"
+
+#include <logging/log.h>
+LOG_MODULE_REGISTER(MODULE, CONFIG_DESKTOP_BAS_LOG_LEVEL);
+
+
+static struct bt_uuid_128 custom_desc_uuid = BT_UUID_INIT_128(
+		0x4c, 0x65, 0xe6, 0xe6, 0x98, 0xb6, 0x78, 0xd5,
+		0x7d, 0x17, 0x19, 0x46, 0x75, 0x2, 0x47, 0x24);
+static const struct bt_uuid_128 custom_desc_chrc_uuid = BT_UUID_INIT_128(
```

Comment:
Can we use the same UUID for service and characteristic ?

Reply:
it should work but as discussed offline I will use different values